WordPress/Classes/ViewRelated/Blog/Blog Details/BlogDetailsViewController+SiteMonitoring.swift

```diff
@@ -0,0 +1,13 @@
+import UIKit
+
+extension BlogDetailsViewController {
+
+    @objc func shouldShowSiteMonitoring() -> Bool {
+        RemoteFeatureFlag.siteMonitoring.enabled() && blog.isAdmin && blog.isAtomic()
```

Comment:
Let's move this to `- (BOOL)supports:(BlogFeature)feature` defined on `Biog`.

Reply:
Done!